features/step_definitions/compose_message_steps.rb

```diff
@@ -0,0 +1,10 @@
+Given(/^I am on the inbox page$/) do
+  visit  mailbox_inbox_path
+end
```

Comment:
Just a heads-up, your colleagues are making the exact same step. Whichever one goes first needs to make sure the other is using the same step (we don't want two steps that do the same thing).